tests/foreman/api/test_discoveryrule.py

```diff
@@ -16,16 +16,35 @@
 
 :Upstream: No
 """
-from fauxfactory import gen_choice, gen_integer
+from fauxfactory import gen_choice, gen_integer, gen_string
 import pytest
 from requests.exceptions import HTTPError
 
 from robottelo.utils.datafactory import valid_data_list
 
 
+@pytest.fixture(scope="module")
```

Comment:
To maintain consistancy through out the files, same applicable for line 33, 39
```suggestion
@pytest.fixture(scope='module')
```